templates/common/on-prem/files/configure-ip-forwarding.yaml

```diff
@@ -40,9 +40,9 @@ contents:
       fi
       echo "${iface}"
     }
-    
+
     iface=$(get_nodeip_hint_interface "${ip_hint_file}")
-    
+
     echo "Node IP interface determined as: ${iface}. Enabling IP forwarding..."
-    sysctl -w net.ipv4.conf.${iface}.forwarding=1
-    sysctl -w net.ipv6.conf.${iface}.forwarding=1
+    echo "1" > /proc/sys/net/ipv4/conf/${iface}/forwarding
+    echo "1" > /proc/sys/net/ipv6/conf/${iface}/forwarding
```

Comment:
please quote `"${iface}"` 

Can interfaces ever contain a `/` ?

Reply:
Done.

`/` and white characters are forbidden from interface names

Reply:
`"` should handle shell meta-characters or globbing.  We could always fuzz.